docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -118,6 +118,13 @@ set to `false` no mapping would get added as when `expand=false` the target mapp
 `expand=true` then the mappings added would be equivalent to `foo, baz => foo, baz` i.e, all mappings other than the
 stop word.
 
+[WARNING]
```

Comment:
Sorry to keep nagging about this - I think this should go under the synonyms_set config (line 47).